### PR TITLE
Add new CSS Structure

### PR DESCRIPTION
**Pull request by annadragan**

### CODE REVIEW

File: Components/utility.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+* {
+  /*box-sizing: inherit;*/
+  box-sizing: border-box;
+}
```
Comment (macfrei) [inline Components/utility.css:4]: This should be in the `base.css` file :)

File: style.css [hunk 2]
```diff
@@ -1,145 +1,78 @@
 @import url("https://fonts.googleapis.com/css2?family=Dancing+Script&family=Handlee&family=Roboto:wght@100&display=swap");
+@import "./Components/base.css";
+@import "./Components/utility.css";
```
Comment (macfrei) [inline style.css]: All the other styles in your `style.css` file could be extracted to their own CSS files and renamed according to BEM.

**Review by macfrei — COMMENTED** [review]
Nice job Anna :)

File: style.css (hunk 2 above)
Comment (annadragan) [reply style.css]: ok, thx a lot!

**Pull request merged**